### ISSUE TITLE
syntax errors not caught in ur pattern and crash tidal

### ISSUE THREAD

**Issue by XiNNiW** [tidalcycles/Tidal#477]
**Describe the bug**
When syntax errors are made in the pattern supplied to ur (the second argument), and the pattern is sufficiently complex, the error is not caught until after the pattern is scheduled to be played and tidal crashes.

**To Reproduce**
1. create a pattern using the `ur` function
2. in the 2nd argument, attempt to apply an effect after euclid (ex. "drums(3,2,1):reverseit")
3. execute the pattern
4. after the error hits the console you are unable to get sound in tidal again

**Expected behaviour**
While it is understandable that the error might halt audio, tidal should recover. It would be helpful if the error were printed but audio not halted. However, I'm not sure how hard that would be to accomplish.

**Environment (please complete the following information):**
 - OS: [e.g. Windows/Mac/Linux]
 - Haskell version: 8.0.2
 - Tidal version: 1.0.7
 - SuperCollider version: 3.10

**Additional context**
Here is some code that causes the problem on my machine:

```
let
mkpat name pattern = (name,pattern)
mkfx name fx = (name,fx)
structure = cat [
    "kicks@8 [kicks,snares]@7 kicks:backrush"
    ,
    "[kicks@3 [kicks@3 kicks(3,8,1):r]]@4 [kicks]@4 [kicks]@7 kicks:r"
  ]
pats = [
    mkpat "kicks" $ sometimes ghost $ s "bd(<4 5 3 6>,16,<0 1 0 3>)"
  ]
fx = [
    mkfx "r" (# speed "-1")
  ]
in
d2 $ ur 16 structure pats fx

```

**Comment by jwaldmann** [comment]
I guess this is #606.

**Comment by yaxu** [comment]
Closed as duplicate of #606. (It shouldn't actually cause a crash, now)
